test/parallel/test-http-agent-keepalive.js

```diff
@@ -81,6 +81,17 @@ function second() {
   }));
 }
 
+function checkSocketKeepAlive(socket) {
+  const symbols = Object.getOwnPropertySymbols(socket);
+  for (let i = 0; i < symbols.length; i++) {
+    const symbol = symbols[i];
+    if (symbol.toString() === 'Symbol(kSetKeepAliveInitialDelay)') {
```

Comment:
I was thinking more of an end-to-end test to ensure that the keepalive settings are in fact being passed as expected. Probably the easiest way to do this is to pass in a custom `createConnection()` and check `options` passed to it.

Reply:
Thanks! I add a new file to test this.